### DISCUSSION TITLE
Service -> Entity로 넘어가는 DTO! entity 측에서는 파라미터 명으로 어떻게 받을까?

### DISCUSSION THREAD

**Discussion by sojukang** [woowacourse-teams/2022-review-duck#496] [Polls]
1. Dto와 create, update 목적을 명시하자! ex) `List<QuestionAnswerUpdateDto> updateDtos`
2. 그걸 entity측에서 알아야할까?  `List<TemplateQuestionUpdateDto> questions`
3. 그 외

여러분의 의견은 어떠십니까?

**Comment by woong7** [comment]
어차피 찝찝할 네이밍이면 아예 명확하게 `questionCreateDto` 같이 명확하게 적자

**Reply by sojukang** [reply]
쓰는 측에서 너무 장황하지 않을까요?
예를 들면 

![image](https://user-images.githubusercontent.com/83529599/193743043-94d8e5b5-09b6-4c3f-a574-e3ec4ded1adb.png)

![image](https://user-images.githubusercontent.com/83529599/193743088-ec23cfd7-19d9-4cc8-a5e8-726d22acb5a3.png)

**Comment by sojukang** [comment]
2번으로 결정되었습니다(Senior Frontend engineer @compy-ryu  의견 적극 반영) 🙇‍♀️.

**Reply by compy-ryu** [reply]
저 3번 찍었는데 제 의견 전혀 반영 안되었어유 ㅡㅡ
